scripts/run.sh

```diff
@@ -6,4 +6,5 @@
 git submodule update --init --recursive
 
 # Invoke CodeQL analysis
-python3 ./src/static/codeql/scripts/codeql.py --clean --builtin
+# If the SCAN_PATH env variable is not set, the argument will be ignored
+python3 ./src/static/codeql/scripts/codeql.py --clean --builtin --scan_path=$SCAN_PATH
```

Comment:
This is nice, Makes it generic to run on any project like oeedger8r or other OE consumers.
We might have to add another suite (oe-codeql-security-queries-only.qls) to scan using codeql queries alone without Open Enclave SDK specific queries.